database_cleanup/models/purge_tables.py

```diff
@@ -64,75 +44,62 @@ def purge(self, cr, uid, ids, context=None):
                 AND confrelid::regclass = '%s'::regclass;
                 """ % line.name)
```

Comment:
done in https://github.com/OCA/server-tools/pull/469/commits/50e09adb9313e04ae9e2f0a0c7a125f468a1e89c